### PR TITLE
updating input message error

### PR DESCRIPTION
**Pull request by JefMari**
for issue #130 

image

@srph @juztinlazaro

### CODE REVIEW

File: styles/base/_form.scss [hunk 1]
```diff
@@ -49,32 +49,31 @@
...
 .form-group .form-control + .message {
+  visibility: hidden;
   position: absolute;
   top: 40px;
   z-index: 9999;
-  visibility: hidden;
   max-width: 220px;
   text-align: center;
+  padding: 10px 15px;
   color: #FFFFFF;
```
Comment (srph) [inline styles/base/_form.scss]: Use `$brand-white`.

Comment (JefMari) [inline styles/base/_form.scss]: ohh didnt see that one

File: styles/base/_form.scss [hunk 2]
```diff
@@ -49,32 +49,31 @@
...
-  padding: 8px 10px;
   background: $brand-crimson;
-  height: auto;
   border-radius: 5px;
+  line-height: 1;
+  font-size: $font-size-small;
 }
```
Comment (srph) [inline styles/base/_form.scss:64]: Can you also fix the rule order? text rules before background / border rules.

Comment (JefMari) [inline styles/base/_form.scss:64]: noted

**Comment by JefMari** [comment]
Heads up!
Already have followup fix!

File: styles/base/_form.scss [hunk 3]
```diff
@@ -49,16 +49,17 @@
...
 .form-group .form-control + .message {
+  visibility: hidden;
   position: absolute;
   top: 40px;
   z-index: 9999;
-  visibility: hidden;
   max-width: 220px;
+  padding: 10px 15px;
+  line-height: 1;
+  font-size: $font-size-small;
```
Comment (srph) [inline styles/base/_form.scss:59]: Nice catch

**Comment by srph** [comment]
Merging this soon. Good to go?

**Comment by JefMari** [comment]
Go go go!

**Pull request merged**